src/worker/ingestion/process-event.ts

```diff
@@ -254,7 +254,28 @@ export class EventsProcessor {
             )
         }
 
-        let updatedProperties: Properties = { ...propertiesOnce, ...personFound.properties, ...properties }
+        // Figure out which properties we are actually setting
+        const returnedProps: Properties = {}
+        let updatedProperties: Properties = {}
```

Comment:
I'd simplify this to:
```js
updatedProperties = { ...personFound.properties } // copy props, will mutate below
```
because then I could remove this more confusing line below:
```js
updatedProperties = { ...personFound.properties, ...updatedProperties }
```

Reply:
Done